### PR TITLE
ADR-755: adds endpoint for duty calculation

### PR DESCRIPTION
**Pull request by msatta**

### CODE REVIEW

File: test/uk/gov/hmrc/alcoholdutycalculator/models/DutyCalculationSpec.scala [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+package uk.gov.hmrc.alcoholdutycalculator.models
+
+import play.api.libs.json.{JsBoolean, JsError, JsNumber, Json}
+import uk.gov.hmrc.alcoholdutycalculator.base.SpecBase
+
+class DutyCalculationSpec extends SpecBase {
+
+  "Volume" when {
+
+    "creating a new Volume" should {
```
Comment (yasaracar) [inline test/uk/gov/hmrc/alcoholdutycalculator/models/DutyCalculationSpec.scala:26]: Can you add tests for the happy case please?

File: test/uk/gov/hmrc/alcoholdutycalculator/services/DutyServiceSpec.scala [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+package uk.gov.hmrc.alcoholdutycalculator.services
+
+import uk.gov.hmrc.alcoholdutycalculator.base.SpecBase
+import uk.gov.hmrc.alcoholdutycalculator.models.{AlcoholByVolume, DutyCalculationRequest, Volume}
+
+class DutyServiceSpec extends SpecBase {
+  val dutyService = new DutyService()
+
+  "dutyService" should {
+    "calculate pure alcohol volume and duty" in {
```
Comment (yasaracar) [inline test/uk/gov/hmrc/alcoholdutycalculator/services/DutyServiceSpec.scala:26]: Can you ad a few more test cases please? Like calculating values without decimals, negative values and a few more rounding cases

File: app/uk/gov/hmrc/alcoholdutycalculator/controllers/DutyCalculationController.scala [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+import uk.gov.hmrc.alcoholdutycalculator.models.DutyCalculationRequest
+import uk.gov.hmrc.alcoholdutycalculator.services.DutyService
+import uk.gov.hmrc.play.bootstrap.backend.controller.BackendController
+
+import javax.inject.Inject
+import uk.gov.hmrc.play.bootstrap.controller.WithJsonBody
+
+import scala.concurrent.Future
+
+class DutyCalculationController @Inject() (
```
Comment (yasaracar) [inline app/uk/gov/hmrc/alcoholdutycalculator/controllers/DutyCalculationController.scala:32]: We had created an integration test for the rates endpoint previously, might be useful to create one for this as well?

**Review by yasaracar — CHANGES_REQUESTED** [review]

**Review by yasaracar — APPROVED** [review]
